ceno_zkvm/src/instructions/riscv/rv32im/mmu.rs

```diff
@@ -48,7 +52,13 @@ impl<E: ExtensionField> MmuConfig<E> {
         io_addrs: &[Addr],
     ) {
         assert!(
-            chain!(static_mem_init.iter_addresses(), io_addrs.iter_addresses()).all_unique(),
+            chain!(
+                static_mem_init.iter_addresses(),
+                io_addrs.iter_addresses(),
+                // TODO: optimize with min_max and Range.
```

Comment:
Why does specifically this one need optimisation?

Reply:
This one is a Range, others are sets.